github/ci/prow-deploy/files/jobs/kubevirt/terraform-provider-kubevirt/terraform-provider-kubevirt-presubmits.yaml

```diff
@@ -0,0 +1,76 @@
+presubmits:
+  kubevirt/terraform-provider-kubevirt:
+  - name: pull-terraform-provider-kubevirt-e2e-k8s-1.20
+    skip_branches:
+      - release-\d+\.\d+
+    annotations:
+      fork-per-release: "true"
+      testgrid-create-test-group: "false"
+    always_run: true
+    optional: false
+    skip_report: false
+    cluster: prow-workloads
+    decorate: true
+    decoration_config:
+      timeout: 1h
+      grace_period: 5m
+    max_concurrency: 11
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror-proxy: "false"
+      preset-shared-images: "true"
+      preset-bazel-cache: "false"
+      preset-bazel-unnested: "false"
+    spec:
+      nodeSelector:
+        type: bare-metal-external
+      containers:
+        - image: quay.io/kubevirtci/bootstrap:v20210311-09ebaa2
+          command:
+            - "/usr/local/bin/runner.sh"
+            - "/bin/sh"
+            - "-c"
+            - "export KUBEVIRT_PROVIDER=k8s-1.20 && automation/test.sh"
```

Comment:
I think it would be better to remove the `export` from here and add an `env` property, something like https://github.com/kubevirt/project-infra/blob/master/github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml#L221-L230

Reply:
Done